### PR TITLE
Feature/story

### PR DESCRIPTION
**Pull request by KrystofVavra**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,9 @@
 		"@gisatcz/ptr-atoms": "^1.6.0",
 		"@gisatcz/ptr-core": "^1.7.0",
 		"@gisatcz/ptr-locales": "^1.5.0",
+		"@gisatcz/ptr-state": "^2.1.0",
```
Comment (vdubr) [inline package.json:36]: You should not add @gisatcz/ptr-state and @gisatcz/visat-components  to this package.

File: src/AppStory/AppStoryBody/content/SidePanel/index.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	const classes = classnames('ptr-AppStorySidePanel', {}, className);
+	const sidePanelRef = useRef();
+
+	return (
+		<div
+			className={classes}
+			ref={sidePanelRef}
+			onScroll={scroll => {
+				onScroll(scroll);
+			}}
```
Comment (vdubr) [inline src/AppStory/AppStoryBody/content/SidePanel/index.js:22]:
```suggestion
			onScroll={onScroll}
```

File: src/AppStory/styles/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+@use 'sass:math';
+@import '~@gisatcz/ptr-core/lib/styles/_variables';
+@import '~@gisatcz/visat-components/lib/styles/_variables';
```
Comment (vdubr) [inline src/AppStory/styles/_variables.scss:3]: Remove dependency on visat-components

File: src/AppStory/styles/index.scss [hunk 4]
```diff
@@ -0,0 +1,149 @@
+@import '~@gisatcz/ptr-core/lib/styles/_variables';
+@import '~@gisatcz/ptr-core';
+@import '~@gisatcz/ptr-components';
+@import '~@gisatcz/ptr-maps';
```
Comment (vdubr) [inline src/AppStory/styles/index.scss:4]: Remove dependency on visat-components, ptr-maps and ptr-components

File: src/AppStory/styles/_variables.scss (hunk 3 above)
Comment (KrystofVavra) [reply src/AppStory/styles/_variables.scss:3]: OK, but I am using its breakpoints, icons and tags. What should I do?

**Pull request closed without merging**